app/(main)/(campaigns)/campaigns.tsx

```
@@ -86,7 +86,12 @@ const Campaigns = () => {
         {Platform.OS === "web" && (
           <View style={styles.campaignsSection}>
             {filteredCampaigns.length > 0 ? (
-              <CampaignsFilledState campaigns={filteredCampaigns} />
+              <FlatList
+                data={[,]}
```

Biome

[error] This array contains an empty slot.

Unsafe fix: Replace hole with undefined


(lint/suspicious/noSparseArray)